src/app/components/universal/newSafetyDocumentUpdate/newSafetyDocumentUpdate.component.html

```diff
@@ -0,0 +1,13 @@
+<div class="container">
+  <div class="row align-items-center">
+    <div class="col-4 text-center">
+      <i class="bi bi-cone-striped"></i>
+    </div>
+    <div class="col-4 text-center">
+      <h5 style="font-weight: bold;"><a href="#" style="text-decoration: none;">Safety Document Updated!</a></h5>
```

Comment:
Unfortunately the atlas standards don't allow (or at least that's the *strong* impression I've gotten from @looksliketyler) for inline styling